pkg/plugins/golang/v4/scaffolds/internal/templates/hack/boilerplate.go

```diff
@@ -106,8 +106,8 @@ Copyright {{ .Year }}.
 {{ index .Licenses .License }}*/`
 
 var knownLicenses = map[string]string{
-	"apache2": apache2,
-	"none":    "",
```

Comment:
@yyy1000, we still need to have the option none 
Also, to keep the same behaviour that we have currently and not change the current default behaviour, we need to ensure that when the license flag is not informed the default value will be apache2.

Reply:
Well, I think it's OK to delete the option 'none'. If we use the option 'none', it will not execute `scaffold.Execute(bpFile)`. So `knownLicenses` will not be used.

Reply:
So, how none can be valid and not exist in knownLicenses[f.License]?